### ISSUE TITLE
viafree.se - Error: No video formats found

### ISSUE THREAD

**Issue by Sopor** [ytdl-org/youtube-dl#24846]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.03.24**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
youtube-dl -f best --all-subs --convert-subs srt https://www.viafree.se/program/samhalle/efterlyst/sasong-55/avsnitt-12 --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-f', 'best', '--all-subs', '--convert-subs', 'srt', 'https://www.viafree.se/program/samhalle/efterlyst/sasong-55/avsnitt-12', '--verbose']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2020.03.24
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.18362
[debug] exe versions: ffmpeg 4.2.1
[debug] Proxy map: {}
[Viafree] avsnitt-12: Downloading webpage
[debug] Using fake IP 78.67.167.112 (SE) as X-Forwarded-For.
[mtg] 1034722: Downloading video JSON
[mtg] 1034722: Downloading streams JSON
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpjwbwqymm\build\youtube_dl\YoutubeDL.py", line 797, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpjwbwqymm\build\youtube_dl\extractor\common.py", line 530, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpjwbwqymm\build\youtube_dl\extractor\tvplay.py", line 311, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpjwbwqymm\build\youtube_dl\extractor\common.py", line 1327, in _sort_formats
youtube_dl.utils.ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

It seems that all videos will generate the same error
`ERROR: No video formats found`

**Comment by Sopor** [comment]
@remitamine Account is only needed if you download from the web browser. If you need an account just let me know where to send it?

**Comment by remitamine** [comment]
fixed with 700265bfcf77f19bdf56062865c1fdf18ca3c9da.

**Comment by Sopor** [comment]
I was going to send you a login but you fixed it anyway ... nice 👍
